test/extensions/bar_chart_data_extensions_test.dart

```diff
@@ -1,43 +1,79 @@
 import 'package:fl_chart/fl_chart.dart';
 import 'package:fl_chart/src/extensions/bar_chart_data_extension.dart';
 import 'package:flutter_test/flutter_test.dart';
+
 import '../chart/data_pool.dart';
 
 void main() {
-  test('test drawRRect', () {
-    expect(
-      MockData.barChartData1
-          .copyWith(alignment: BarChartAlignment.start)
-          .calculateGroupsX(100),
-      [9.0, 43.0, 77.0],
-    );
-
-    expect(
-      MockData.barChartData1
-          .copyWith(alignment: BarChartAlignment.end)
-          .calculateGroupsX(100),
-      [23.0, 57.0, 91.0],
-    );
-
-    expect(
-      MockData.barChartData1
-          .copyWith(alignment: BarChartAlignment.center)
-          .calculateGroupsX(100),
-      [16.0, 50.0, 84.0],
-    );
-
-    expect(
-      MockData.barChartData1
-          .copyWith(alignment: BarChartAlignment.spaceBetween)
-          .calculateGroupsX(100),
-      [9.0, 50.0, 91.0],
-    );
-
-    expect(
-      MockData.barChartData1
-          .copyWith(alignment: BarChartAlignment.spaceAround)
-          .calculateGroupsX(100),
-      [16.666666666666668, 50.0, 83.33333333333334],
-    );
+  group('BarChartDataExtension.calculateGroupsX', () {
```

Comment:
I put this into a group, so we can see the method under test in the tests' output

Reply:
Sounds good!